<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntraPoolCopyPage</class>
 <widget class="QWizardPage" name="IntraPoolCopyPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Copy VM</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelRubric">
     <property name="text">
      <string>Enter a name and description for the copy of the VM:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label2">
       <property name="text">
        <string>&amp;Name:</string>
       </property>
       <property name="buddy">
        <cstring>NameTextBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="NameTextBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label1">
       <property name="text">
        <string>&amp;Description:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="buddy">
        <cstring>DescriptionTextBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="DescriptionTextBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="DecentGroupBox" name="groupBox1">
     <property name="title">
      <string>Copy mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="ToolTipContainer" name="toolTipContainer1" native="true"/>
      </item>
      <item>
       <widget class="QRadioButton" name="CopyRadioButton">
        <property name="text">
         <string>Full c&amp;opy</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="tableLayoutPanelSrPicker">
        <item>
         <widget class="QLabel" name="labelSrHint">
          <property name="text">
           <string>Select the storage repository for the full copy:</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="SrPicker" name="srPicker1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonRescan">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&amp;Rescan</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacerBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ToolTipContainer</class>
   <extends>QWidget</extends>
   <header>controls/tooltipcontainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DecentGroupBox</class>
   <extends>QGroupBox</extends>
   <header>controls/decentgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SrPicker</class>
   <extends>QWidget</extends>
   <header>controls/srpicker.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
